### PR TITLE
[gfx11][Solvers][Winograd] ConvWinoFuryRxSFused (convolution+bias+activation)

### PR DESCRIPTION
**Pull request by averinevg**
ConvWinoFuryRxSFused v2.4 (gfx11, NCHW, FP16)

### CODE REVIEW

**Review by junliume — APPROVED** [review]

**Comment by apwojcik** [comment]
@junliume @averinevg The PR broke compilation on Windows. Please fix it. The compilation of tests `cba_infer` and `cba_find2_infer` is broken on Windows. From now on, all PRs will not compile on Windows.

File: src/include/miopen/fusion/solvers.hpp [hunk 1]
```diff
@@ -277,6 +277,40 @@ struct ConvBinWinogradRxSf2x3g1Fused final : FusionSolverBase
     GetSolution(const FusionContext& context, const FusionDescription& problem) const override;
 };
 
+template <uint32_t Winodata, uint32_t Winofilter>
+struct ConvWinoFuryRxSFused final : FusionSolverBase
+{
+    const std::string& SolverDbId() const override
```
Comment (apwojcik) [inline src/include/miopen/fusion/solvers.hpp:283]: Add `MIOPEN_INTERNALS_EXPORT` here. It is required for explicit template instantiation below (line 307).

File: src/include/miopen/fusion/solvers.hpp [hunk 2]
```diff
@@ -277,6 +277,40 @@ struct ConvBinWinogradRxSf2x3g1Fused final : FusionSolverBase
...
+    {
+        return GetSolverDbId<ConvWinoFuryRxSFused<Winodata, Winofilter>>();
+    }
+
+    MIOPEN_INTERNALS_EXPORT bool IsApplicable(const FusionContext&,
+                                              const FusionDescription&) const override;
+    bool IsDynamic() const override { return true; }
```
Comment (apwojcik) [inline src/include/miopen/fusion/solvers.hpp:290]: Add `MIOPEN_INTERNALS_EXPORT` here. It is required for explicit template instantiation below (line 307).

File: src/include/miopen/fusion/solvers.hpp (hunk 1 above)
Comment (averinevg) [reply src/include/miopen/fusion/solvers.hpp:283]: Why is this not required for other solvers?

Comment (apwojcik) [reply src/include/miopen/fusion/solvers.hpp:283]: This templated solver structure is the only one fully instantiated at line 307. The rest of the solvers are partially instantiated templated structures.

**Pull request merged**